addons/godot_rl_agents/controller/ai_controller_2d.gd

```diff
@@ -1,9 +1,17 @@
 extends Node2D
 class_name AIController2D
 
-enum ControlModes { INHERIT_FROM_SYNC, HUMAN, TRAINING, ONNX_INFERENCE, RECORD_EXPERT_DEMOS }
+enum ControlModes {
+    INHERIT_FROM_SYNC, ## Inherit setting from sync node
+    HUMAN, ## Test the environment manually
+    TRAINING, ## Train a model
+    ONNX_INFERENCE, ## Load a pretrained model using an .onnx file
+    RECORD_EXPERT_DEMOS
```

Comment:
Not sure what RECORD_EXPERT_DEMOS does so I don't have a description here.

Reply:
It's an option for recording expert demonstrations (i.e. playing the game manually to show the agent what to do). The data includes the obs and actions taken. However, a bit more code needs to be implemented when using this option.

There is a wip course bonus unit for this: https://github.com/huggingface/deep-rl-class/pull/537

Reply:
Thanks, I added the description.